### DISCUSSION TITLE
Is the capture in the proxy mode of Freerdp2.10.0 unavailable?

### DISCUSSION THREAD

**Discussion by Mr-LZG** [FreeRDP/FreeRDP#8982] [Q&A]
Is the capture in the proxy mode of Freerdp2.10.0 unavailable? After I configure it, I can only see that the directory is created, but the directory is always empty.

**Comment by akallabeth** [comment]
looks like you answered it (mostly) yourself with https://github.com/FreeRDP/FreeRDP/discussions/8983 (also, please do not double post)

**Reply by Mr-LZG** [reply]
After "session captrue" is enabled, unsupported GFX clients are not allowed to connect。
![image](https://github.com/FreeRDP/FreeRDP/assets/22317592/add2026b-499f-473a-94cf-6f256f5c25bb)

So, #8983  doesn't answer my question

**Reply by akallabeth** [reply]
so, what is your question then?

**Reply by Mr-LZG** [reply]
After the session ends, the directory is empty and "session capture" doesn't capture anything.

**Reply by akallabeth** [reply]
still not a question, but an observation of something (I don´t really know what)
sorry, can´t help here if you are so secretive.

**Reply by Mr-LZG** [reply]
What information do I need to provide

**Reply by Mr-LZG** [reply]
My question is: Although "session capture" is enabled, the test found that only a directory was created, but nothing was generated in the directory. It feels that the "session capture" function has not been developed, and I have not found the relevant code.

**Reply by akallabeth** [reply]
so, you:
1. successfully established a connection over the proxy?
2. after this session disconnected you don´t have anything in that folder?

sorry, your descriptions are really short as well as ambigous and it is hard to guess what exactly you are asking.

**Reply by Mr-LZG** [reply]
1.  Yes, successfully connected. 
2.  There is nothing in the folder when the session is disconnected

**Reply by akallabeth** [reply]
@Mr-LZG ok, did have a look at the capture plugin (did not write that one, so new to it as well ;))
it writes the data to a `socket` and not to a file? 
can you tell how you tested?
